public/pages/Destinations/containers/CreateDestination/ManageSenders/ManageSenders.js

```diff
@@ -263,9 +267,9 @@ export default class ManageSenders extends React.Component {
                   name="senders"
                   validateOnChange={true}
                   render={(arrayHelpers) =>
-                    loadingSenders ? (
+                    !isEmailAllowed || loadingSenders ? (
```

Comment:
nit: remove !isEmailAllowed and just set loadingSenders to false in the componentDidMount if email isn't allowed.

Reply:
`loadingSenders` will eventually be false even when email is allowed. This check was made so we display the message in the `div` instead of calling `renderSenders` when email is not allowed. Otherwise this will eventually show the empty senders message when `renderSenders` is called since it can't retrieve the email senders/groups which is confusing behavior to the user.